src/styles/partials/components/_alt-list.scss

```diff
@@ -0,0 +1,56 @@
+ol.dg_alt-list {
+  list-style-type: none;
+  margin-bottom: $margin-large;
+  margin-left: 0;
+  padding-left: 0;
+  text-decoration: none;
+}
+
+ul.dg_alt-list {
+  list-style-type: none;
+  margin-bottom: $margin-large;
+  margin-left: 0;
+  padding-left: 0;
+  text-decoration: none;
+}
+
+.dg_i-float-right {
+  color: $gray-light;
+  float: right;
+  font-size: $large-icon-size;
+}
+
+.dg_alt-list {
+  li {
+    border-bottom: $border-width-small solid $gray-light;
+    margin-top: $default-margin;
+    -webkit-transition: all 0.3s ease 0s;
```

Comment:
i don't think we want to transition everything, we will only want to transition what already is handled by the a and button transitions. You should also check out the _animations.scss mixins file. You might have questions, if so, please let me know.

Reply:
Taking a look at what exists in the _animations.scss file and will follow up with any questions.

Reply:
Updating with the mixin and will make a commit now.